code/modules/overmap/internet/internet_repeater.dm

```diff
@@ -11,6 +11,7 @@ var/global/list/internet_repeaters = list()
 	use_power = POWER_USE_ACTIVE
 	idle_power_usage = 50
 	active_power_usage = 5000
+	construct_state = /decl/machine_construction/default/panel_closed
```

Comment:
The current circuitboard for it gives it a (nonbuildable!) terminal on spawn, which seems clearly wrong (note that it also has the default nonbuildable apc teslalink component as well so you won't notice). Please double-check the components.